app/src/pages/project/RetrievalEvaluationLabel.tsx

```diff
@@ -1,15 +1,19 @@
 import React, { useMemo } from "react";
+import { Color } from "three";
 import { css } from "@emotion/react";
 
 import {
   Flex,
+  HelpTooltip,
```

Comment:
didn't do a ton of moving (to components or elsewhere) or changing to this component since i wasn't sure how long lived it would be, i.e., will we just merge this in with AnnotationLabel at some point? Can move to components if needed